libpod/info.go

```diff
@@ -127,6 +128,11 @@ func (r *Runtime) hostInfo() (*define.HostInfo, error) {
 		SwapFree:           mi.SwapFree,
 		SwapTotal:          mi.SwapTotal,
 	}
+	platform := parse.DefaultPlatform()
```

Comment:
./vendor/github.com/google/go-containerregistry/pkg/v1/platform.go
Parses the platform string.
Should we care about the OS and Arch?

Reply:
The runtime Go library gives you the OS and Arch with `runtime.GOOS` and `runtime.GOARCH`. That is what is already used here but it doesn't give you the variant, so using `parse.DefaultPlatform` to get the variant information.